### PR TITLE
Adding optional billing profile to VMSS Create and Update APIs for Api Version 2019-03-01

### PR DESCRIPTION
**Pull request by rakeshakkera**
…ersion 2019-03-01

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 1]
```diff
@@ -6208,6 +6208,14 @@
       },
       "description": "Specifies the boot diagnostic settings state. <br><br>Minimum api-version: 2015-06-15."
     },
+    "BillingProfile": {
+      "maxPrice": {
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: move this property to inside of "properties":

Comment (rakeshakkera) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Thank you for pointing out. This was causing the error initially. Made the change.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 2]
```diff
@@ -7726,6 +7736,10 @@
             "name": "VirtualMachineEvictionPolicyTypes",
             "modelAsString": true
           }
+        },
+        "billingProfile": {
+          "$ref": "#/definitions/BillingProfile",
+          "description": "Specifies the billing related details of a low priority VMSS. <br><br>Minimum api-version: 2019-03-01."
```
Comment (zikalino) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: I wonder if we need description here. It seems exactly the same as in referred type definition.

**Review by zikalino — APPROVED** [review]
looks good!

**Comment by zikalino** [comment]
@hyonholee is it ok to merge?

**Pull request closed without merging**